source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -0,0 +1,45 @@
+.. _backend_20220519:
+
+19 May 2022 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Introduced option to encode HTTPS Endpoint responses as :ref:`EJSON or JSON <endpoint-return-types>`.
+- Added support for serverless Atlas instances as data sources. Serverless instances do not yet
```

Comment:
Do you know if we need to do anything special to use a serverless instance vs. a regular instance? I wonder if there are any docs we can/should link to, or if there's no change at all, then probably not.

Reply:
I couldn't find anything here, will reach out and see if I should link to anything. I suspect you select it like any other atlas instance, so maybe there really wasn't anything to doc?